install.txt

```diff
@@ -0,0 +1,30 @@
+Installation Instructions: 
+
+The directory corresponding to your cart's version will be called 
+YOUR_VERSION.  For example, if you are running Zen Cart 1.5.5, 
+YOUR_VERSION is 1_5_5. 
+
+1) In includes, change the directory name YOUR_TEMPLATE 
+to correspond to what's being used in your cart.
+Then copy all the files in this directory to your cart.
+
+2) Set the template names in YOUR_VERSION:
+
+a) If you are using pre-155 OR a non-responsive template, 
+change the directory name YOUR_TEMPLATE to correspond to 
+what's being used in your cart, then remove YOUR_CLASSIC_RESPONSIVE.
+
+b) If you are using 155 with a responsive template, 
+change the directory name YOUR_CLASSIC_RESPONSIVE to correspond to 
+what's being used in your cart, then remove YOUR_TEMPLATE.
+
+Then copy all the files in this directory to your cart.
+
+3) In admin, copy all files to your cart.
+
+4) Login to your cart's admin panel, and browse to 
```

Comment:
Unfortunately at this point the configuration option for SBA has not been loaded. Making it available requires entering the filename on the browser. I haven't dug in to determine if this is on purpose or just wasn't coded to happen. It has been documented though in potteryhouse's instructions found under his github repo section of https://github.com/potteryhouse/stock_by_attribute_1.5.X_Instructions that I have also forked to my account.